main.tf

```diff
@@ -1,16 +1,149 @@
+module "lambda_package" {
+  source  = "matti/resource/shell"
+  version = "~>1.0.7"
 
-module "tags" {
-  source  = "rhythmictech/tags/terraform"
-  version = "1.0.0"
+  # I want to be using `npm run-script build` here,
+  # but for some reason I can't work out `node-lambda` fails in that context
+  command     = "node-lambda package -e production -x 'event.json build.env bin .github' -I lambci/lambda:build-nodejs12.x"
```

Comment:
This isn't sufficiently idiot proof. It does not seem to respect the credential process and also I had to figure out that I needed to install node-lambda via npm. So need some README updates at a minimum, but I'm also concerned about the fact that I need AWS creds just to _build_ the thing?

Reply:
wait it makes you set AWS creds to build? uuuuuuuh....

I'll probably set up an alternative build process then.

Reply:
I like that terraform is extensible but if we have to drop to the shell in our "declarative" IAC... maybe we should be rethinking our tooling.